x11-packages/tinysparql/build.sh

```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://gnome.pages.gitlab.gnome.org/tinysparql
+TERMUX_PKG_DESCRIPTION="Low-footprint RDF triple store with SPARQL 1.1 interface"
+TERMUX_PKG_LICENSE="LGPL-2.1-or-later"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=3.8.beta
```

Comment:
Why ship a beta version?
Also that's not the current version.
```suggestion
TERMUX_PKG_VERSION=3.9.2
```

Reply:
it was from when i was porting nautilus, sorry i really should have checked what's latest